src/main/java/redis/clients/jedis/JedisClusterInfoCache.java

```diff
@@ -105,7 +109,37 @@ public void setNodeIfNotExist(HostAndPort node) {
       if (nodes.containsKey(nodeKey)) return;
 
       JedisPool nodePool = new JedisPool(poolConfig, node.getHost(), node.getPort(),
-          connectionTimeout, soTimeout, null, 0, null);
+          connectionTimeout, soTimeout, null, 0, null, false, null, null, null);
```

Comment:
You could replace this method body with:
`setNodeIfNotExist(node, false, null, null, null)`